plot-lib/plot/private/plot2d/plot-area.rkt

```diff
@@ -672,13 +675,20 @@
 
     (define/public (end-renderers)
       (clear-clip-rect)
-      (send pd reset-drawing-params))
+      (send pd reset-drawing-params)
+      (when (and (not (empty? legend)) (inside-anchor (plot-legend-anchor)))
+        (draw-legend legend)))
 
     (define/public (draw-legend legend-entries)
       (define gap-size (+ (pen-gap) tick-radius))
+      (define-values (x-min x-max y-min y-max)
+        (if (outside-anchor (plot-legend-anchor))
+            (values dc-x-min (+ dc-x-min dc-x-size)
+                    dc-y-min (+ dc-y-min dc-y-size))
```

Comment:
Do I understand correctly that, if the legend is "outside" you anchor it on the corners of the entire drawingarea?

Reply:
This PR doesn't really finish implementing the 'outside functionality. For now it indeed just uses the outside bounds of the entire drawing area and places it relative to that. the last PR (#74) then made the actual space reservation and used better bounds as to not draw over the title. When I made this PR I was anticipating more work in that PR, and since we later split out #75 and #76, There are even less changes in 74...